test/test_integration_transcoders.cxx

```diff
@@ -466,6 +468,47 @@ TEST_CASE("integration: subdoc with public API", "[integration]")
         REQUIRE(resp.content_as<std::uint32_t>(couchbase::subdoc::lookup_in_macro::value_size_bytes) == 66);
     }
 
+    {
+        auto [ctx, resp] = collection
+                             .mutate_in(id,
+                                        couchbase::mutate_in_specs{
+                                          couchbase::mutate_in_specs::increment("views", 1).create_path(),
+                                          couchbase::mutate_in_specs::remove("missing_field"),
+                                        },
+                                        {})
+                             .get();
+        REQUIRE(ctx.ec() == couchbase::errc::key_value::path_not_found);
+        REQUIRE(ctx.first_error_index().has_value());
+        REQUIRE(ctx.first_error_index().value() == 1);
+        REQUIRE(ctx.first_error_path().has_value());
+        REQUIRE(ctx.first_error_path() == "missing_field");
+        REQUIRE(resp.cas().empty());
+        REQUIRE_THROWS_WITH(resp.has_value(0), Contains("path_invalid"));
+        REQUIRE_THROWS_WITH(resp.has_value("views"), Contains("path_invalid"));
+        REQUIRE_THROWS_WITH(resp.content_as<std::uint32_t>(0), Contains("path_invalid"));
+        REQUIRE_THROWS_WITH(resp.content_as<std::uint32_t>("views"), Contains("path_invalid"));
+    }
+
+    {
+        auto [ctx, resp] = collection
+                             .mutate_in(id,
+                                        couchbase::mutate_in_specs{
+                                          couchbase::mutate_in_specs::increment("views", 1).create_path(),
+                                          couchbase::mutate_in_specs::upsert("references", 100'500).create_path(),
+                                        },
+                                        {})
+                             .get();
+        REQUIRE_SUCCESS(ctx.ec());
```

Comment:
It depends on #262 here, I will rebase it once that PR will be merged.

This one still ready and good for review.